workshops/views.py

```diff
@@ -917,6 +918,55 @@ def export(request, name):
 
 #------------------------------------------------------------
 
+@login_required
+def workshops_over_time(request):
+    '''Export CSV of count of workshops vs. time.'''
+
+    data = Counter()
+    for e in Event.objects.past_events():
+        data[e.start.isoformat()] += 1
+    return _time_series(request, data, 'Workshop over time')
+
+
+@login_required
+def learners_over_time(request):
+    '''Export CSV of count of learners vs. time.'''
+
+    data = Counter()
+    for e in Event.objects.past_events():
+        data[e.start.isoformat()] += e.attendance \
+                                     if e.attendance is not None \
```

Comment:
Actually it doesn't matter if we `order_by`, because `dict` is unordered anyway.